unity-renderer/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogUrlUtils/AssetCatalogUrlsUtils.cs

```diff
@@ -1,24 +1,30 @@
 using DCL.Configuration;
 
-public static class BIWUrlUtils
+public static class AssetCatalogUrlsUtils
```

Comment:
The name is a bit unreadable, I like `AssetCatalogUtils` for clarity, but it's on you.